magento/pwa-studio
# fix: center button on communications page

# Description

Small mobile css fix to center button.

## Related Issue





Closes [PWA-846](https://jira.corp.magento.com/browse/PWA-846).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Sign in
2. Go to communication page.
3. Verify button is left aligned on desktop and center aligned on mobile.

## Screenshots / Screen Captures (if appropriate)

[![Image from Gyazo](https://i.gyazo.com/04ca455917f2fae4923743464a63fadb.png)](https://gyazo.com/04ca455917f2fae4923743464a63fadb)

## Checklist

* I have updated the documentation accordingly, if necessary.
